examples/large_models/Huggingface_accelerate/llama2/custom_handler.py

```diff
@@ -0,0 +1,135 @@
+import logging
+from abc import ABC
+
+import torch
+import transformers
+from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer, BitsAndBytesConfig
+from accelerate import init_empty_weights
+from accelerate import load_checkpoint_and_dispatch
+
+from ts.context import Context
+from ts.handler_utils.distributed.deepspeed import get_ds_engine
+from ts.torch_handler.distributed.base_deepspeed_handler import BaseDeepSpeedHandler
+
+logger = logging.getLogger(__name__)
+logger.info("Transformers version %s", transformers.__version__)
+
+
+class TransformersSeqClassifierHandler(BaseDeepSpeedHandler, ABC):
```

Comment:
@lxning any particular reason extending `BaseDeepSpeedHandler`?

Reply:
changed it to BaseHandler